src/scss/components/statistics.scss

```diff
@@ -0,0 +1,125 @@
+.statistics {
+  position: relative;
+  display: block;
+  z-index: 1;
+  overflow: hidden;
+
+  &__container {
+    padding-top: 12rem;
+    padding-bottom: 12rem;
+    background-color: var(--brand-navy);
+
+    @include breakpoint(md) {
+      padding-top: 15.8rem;
+      padding-bottom: 15.8rem;
+    }
+  }
+
+  &__headlines {
+    @include grid;
+  }
+
+  &__eyebrow {
+    grid-column: 1/9;
```

Comment:
For all the grid columns please use the mixin created in the grid.scss i.e (@include col(4,6); intead of grid-column 1/9).

Reply:
Done ! thanks!